pkg/rpc/response/result/invoke.go

```diff
@@ -57,7 +57,7 @@ func (r Invoke) MarshalJSON() ([]byte, error) {
 
 	var tx string
 	if r.Transaction != nil {
-		st := hex.EncodeToString(r.Transaction)
+		st := base64.StdEncoding.EncodeToString(r.Transaction)
```

Comment:
I think we can declare `Transaction` as `[]byte` now in `invokeAux` and simplify things a bit.